modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -1,6 +1,7 @@
 import { Arr, Fun, Obj, Optionals, Strings, Type } from '@ephox/katamari';
 import { Attribute, Class, ContentEditable, Css, Html, Insert, Remove, Selectors, SugarElement, SugarNode, Traverse, WindowVisualViewport } from '@ephox/sugar';
 
+import Schema from '../../api/html/Schema';
```

Comment:
um this is the same target file it was before, just with an extra `../api` in the middle, please put it back

Reply:
done, sorry